src/widgets/hits/hits.js

```diff
@@ -41,7 +44,11 @@ function hits({
     hitsPerPage = 20
   } = {}) {
   if (!container) {
-    throw new Error(usage);
+    throw new Error('Must provide a container.' + usage);
+  }
+
+  if (templates.item && templates.items) {
+    throw new Error('Must contain only items OR item template.' + usage);
```

Comment:
I am good with allItems thanks for the explanation. We just need to fix the small places where you said "items" and we are good (usage, throw)